camel/toolkits/github_toolkit.py

```diff
@@ -71,6 +72,63 @@ def summary(self) -> str:
         )
 
 
+@dataclass
+class GithubPullRequestDiff:
```

Comment:
nit:
```suggestion
class GitHubPullRequestDiff:
```

Reply:
I'm not super sure about this one, cause I tried to follow the same naming style as https://github.com/PyGithub/PyGithub which we use underneath and they follow lowercase "h"